app/serializers/concept_result_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ConceptResultSerializer < ActiveModel::Serializer
+  attributes :concept_name, :metadata
+
+  def concept_name
```

Comment:
@kriskelly how about adding a line `belongs_to :concept` under the `attributes` line, this way concept name can be fetched with concept_result.concept.name ?